.vscode/tasks.json

```diff
@@ -0,0 +1,69 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "cppbuild",
+			"label": "Debug C/C++: CUDA Makefile",
+			"command": "make",
+			"args": [
+				"debug",
+			],
+			"options": {
+				"cwd": "${workspaceFolder}/build_cuda"
+			},
+			"problemMatcher": [
+				"$gcc"
+			],
+			"group": "build",
+			"detail": "compiler: /opt/slurm/intel/oneapi/2024.0.0.49397/compiler/2024.0/bin/icpx"
```

Comment:
Will this need to be updated for each new oneAPI release? Is this a file that can be generated by VSCode itself?